app/controllers/projects_controller.rb

```diff
@@ -16,12 +17,13 @@ def show
     @authors       = [current_user]
     @boards        = current_project.methodology_library.boards
     @issues        = current_project.issues.includes(:tags).sort
-    @methodologies = current_project.methodology_library.notes.map{|n| Methodology.new(filename: n.id, content: n.text)}
+    @methodologies = current_project.methodology_library.notes.map { |n| Methodology.new(filename: n.id, content: n.text) }
     @nodes         = current_project.nodes.in_tree
     @tags          = current_project.tags
+    @tasks         = assigned_cards(current_project.id)
```

Comment:
This approach is a bit backwards and inconsistent w/ what we have for other records.

Also, we typically go from the project to the nested resources at the controllers, we don't delegate that searchto the view